perun-rpc/src/main/java/cz/metacentrum/perun/rpc/Api.java

```diff
@@ -255,8 +260,11 @@ else if (isNotEmpty(req.getHeader(OIDC_CLAIM_SUB))) {
 			} else {
 				throw new InternalErrorException("OIDC issuer not send by Authorization Server");
 			}
-			extSourceLoaString = "-1";
-			log.debug("detected OIDC/OAuth2 client for sub={},iss={}",extLogin,iss);
+			var oidcSubIssuer = initOidcAuth(req, additionalInformations);
+			extLogin = oidcSubIssuer.sub;
+			extSourceName = oidcSubIssuer.issuer;
+			extSourceLoaString = "3";
```

Comment:
this is something to discuss, are we okay with this? or will we also getting this information from user info?